src/Symfony/Bundle/FrameworkBundle/CacheWarmer/ValidatorCacheWarmer.php

```diff
@@ -70,6 +75,19 @@ protected function warmUpPhpArrayAdapter(PhpArrayAdapter $phpArrayAdapter, array
         return parent::warmUpPhpArrayAdapter($phpArrayAdapter, $values);
     }
 
+    protected function getPhpArrayFile(string $cacheDir, ?string $buildDir = null): ?string
+    {
+        if (!$buildDir) {
+            return null;
+        }
+
+        if (Path::isRelative($this->phpArrayFile)) {
```

Comment:
I'm not convinced we need this logic, it adds complexity IMHO

Reply:
Removed